src/internal/tripletex-time-agent.ts

```diff
@@ -8,6 +8,16 @@ import { Config } from '@pulumi/pulumi';
 
 const config = new Config('tripletex-time-agent');
 
+export const setup = new ProjectOnGithub(
+  'tripletex-time-agent',
+  {
+    projectName: 'tripletex-time-agent',
+    folderId: folder.id,
+    repository: 'tripletex-time-agent',
+  },
+  { providers: [bjerkio] },
+);
+
```

Comment:
Is this change relevant?

Reply:
No, but it was stopping the build from working, so I change that too